### PR TITLE
Crossword — Remove misleading app link for crosswords on Desktop

### PR DESCRIPTION
**Pull request by mchv**
## What is the value of this and can you measure success?

This is a follow-up from #27483 and #22331. We are currently misleading our desktop users to download our mobile app, which can't be downloaded. 

Screenshot 2024-10-18 at 14 07 54

## What does this change?

Remove noise for our desktop users.

## Screenshots

**Before changes on desktop**

Screenshot 2024-10-18 at 15 01 17

**After changes on desktop**

Screenshot 2024-10-18 at 15 02 44

### CODE REVIEW

**Review by bryophyta — APPROVED** [review]
I'm not on the WebX team at the moment so can't approve as a codeowner but this looks good to me!

**Review by DanielCliftonGuardian — APPROVED** [review]

**Pull request merged**
